tensorflow/lite/micro/memory_helpers_test.cc

```diff
@@ -111,6 +117,11 @@ TF_LITE_MICRO_TEST(TestAlignSizeUp) {
   TF_LITE_MICRO_EXPECT_EQ(static_cast<size_t>(24), tflite::AlignSizeUp(23, 2));
   TF_LITE_MICRO_EXPECT_EQ(static_cast<size_t>(24), tflite::AlignSizeUp(23, 3));
   TF_LITE_MICRO_EXPECT_EQ(static_cast<size_t>(32), tflite::AlignSizeUp(23, 16));
+
+  TF_LITE_MICRO_EXPECT_EQ(static_cast<size_t>(32),
+                          tflite::AlignSizeUp<TestAlignSizeUp>());
+  TF_LITE_MICRO_EXPECT_EQ(static_cast<size_t>(64),
+                          tflite::AlignSizeUp<TestAlignSizeUp>(2));
```

Comment:
Let's have a separate test case for the templatized align size up.

Reply:
Done